specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/virtualMachineScaleSet.json

```diff
@@ -5391,6 +5391,10 @@
           },
           "description": "The virtual machine child extension resources."
         },
+        "identity": {
```

Comment:
please place this new property in the last place of the properties list.  so after 'zones'